docs/DesignDocs/CustomAttestation.md

```diff
@@ -927,9 +965,24 @@ size_t params_size = sizeof(params);
 oe_claim_t claims = { ... };
 size_t claims_size = ...;
 
+/* Receive the evidence format ids that the verifier supports */
+recv(VERIFIER_SOCKET_FD, evidence_format_ids, evidence_format_id_length, 0);
+
+/* Get registered attester format ids and find a common format */
+oe_get_registered_attester_format_ids(*format_ids, &format_ids_length);
+for (size_t m = 0; m < format_ids_length; m++)
+{
+    for (size_t n = 0; n < evidence_format_id_length; n++)
```

Comment:
I would add braces around the for loop for better clarity like 'for (...) { if (...) {} }`